regen/embed.pl

```diff
@@ -720,4 +759,4 @@ sub update_headers {
 
 update_headers() unless caller;
 
-# ex: set ts=8 sts=4 sw=4 noet:
+# ex: set ts=8 sts=4 sw=4 et:
```

Comment:
Umm  ...  ??? Why was this sent thru astyle or clangformat?

Reply:
I don't understand this comment.  It was not sent through those things.  The commit message for the commit that changes it explains why it was done